bin/greatEscape.cpp

```diff
@@ -105,13 +111,19 @@ void updateGame(GameLogic &gameLogic, MenuView &menuView, GameView &gameView){
 			break;
 
 		case GameState::State::SUCCESS:
+
+			timer.stopClock();
+
 			if (gameLogic.state.getCurrentLevel() == gameLogic.state.getUnlockedLevels()){
 				gameLogic.state.incrementUnlockedLevels();
 			}
 			gameLogic.state.setState(GameState::State::LEVELSELECT);
 			break;
 
 		case GameState::State::FAIL:
+
+			timer.stopClock();
```

Comment:
Same thing, timer should stop when we detect failure not when we're on the fail page